### ISSUE TITLE
[Proposal] Show new notifications only within the taskbar

### ISSUE THREAD

**Issue by Zeuselhood** [bkaankose/Wino-Mail#118]
As many here, my goal is to use WinoMail as a windows mail app replacement. 

Most things that make it suitable for me have already been implemented.
However, there is one thing that the windows mail client does different:

On the windows taskbar, you have **notification count** of how many new E-Mails you have.
The windows mail client only counts the new mails after the app has been started the last time (or last synchronised?), while WinoMail's counter includes every unread notification.

I don't open every mail, sometimes i will go through them after a few months and I like how you can "start your day" and see every new mail while you working with the PC with the windows mail client. For example, I have currently one new mail on the task bar badge of the windows mail client (so i know i got one mail in the recent hours) while I have 153 mails on the WinoMail badge, which are all my unread mails.

The best implementation would probably be an option for the user to choose because its a individual preferation. If only one way can be implemented, I would go with  the windows mail client way, as WinoMail will be a replacment for many.

**Keep up the great work!**

**Comment by bkaankose** [comment]
It's a fair point. I agree with you. What behavior options would you like to see exactly?

Update the badge based on:

1) Unread e-mails for all accounts' Inbox folder. (this is current behavior)
2) Unread e-mails for recent messages. (Recent meaning that mails that arrived today)
3) Something else?

I'm open for ideas.

**Comment by Zeuselhood** [comment]
Yes, I think these are the two options (nor sure if there should be more, or at least none that I can think of). But the 2. option would also be for ALL inboxes, the wording makes it sound like that's only the case for the 1st one. But you probably just wrote it down and the wording will change anyway.Am 21.03.2024 06:48 schrieb Burak Kaan Köse ***@***.***>:
It's a fair point. I agree with you. What behavior options would you like to see exactly?
Update the badge based on:
Unread e-mails for all accounts' Inbox folder. (this is current behavior)Unread e-mails for recent messages. (Recent meaning that mails that arrived today)Something else?
I'm open for ideas.

**Comment by Zeuselhood** [comment]
I think Windows Mail does it session wise. Here is an example:

- I start my computer and open Windows Mail and the taskbar shows no unread messages, eventhough i have a few hundred.
- I minimize windows Mail into the background and use another app, a browser for example.
- I get one new Mail and the taskbar badge says "1" so I now for my current session i have 1 new message
- I open Winowsmail and read the mail. Maybe I set it to unread again since i want to work on that later. 
- However, the taskbar counter jumps to "0" (-> dissapears) since it nows i have noticed the mail.
- > Everytime the mail app is opened it resets the notification count.

For me, this behavior is perfect since you can keep track of incoming mails while you are working on your laptop, while you can still keep many apps on unread to go thorugh them maybe once a month. I also like it more then completly turn the badge of obviously.

**Comment by bkaankose** [comment]
> I think Windows Mail does it session wise. Here is an example:
> 
> * I start my computer and open Windows Mail and the taskbar shows no unread messages, eventhough i have a few hundred.
> * I minimize windows Mail into the background and use another app, a browser for example.
> * I get one new Mail and the taskbar badge says "1" so I now for my current session i have 1 new message
> * I open Winowsmail and read the mail. Maybe I set it to unread again since i want to work on that later.
> * However, the taskbar counter jumps to "0" (-> dissapears) since it nows i have noticed the mail.
> * > Everytime the mail app is opened it resets the notification count.
> 
> For me, this behavior is perfect since you can keep track of incoming mails while you are working on your laptop, while you can still keep many apps on unread to go thorugh them maybe once a month. I also like it more then completly turn the badge of obviously.

This means that they only count "**unread messages from the last synchronization date**". Makes sense.

**Comment by Zeuselhood** [comment]
> This means that they only count "**unread messages from the last synchronization date**". Makes sense.

Yes probably something like that.
Are you planning to add that? 

Maybe it's even possible to let the user decide how he wants the notification badge to behave. 
1. Show all Unread Emails
2. Show new emails since last sync 
3. Show no badge

**Comment by ugogstetter** [comment]
I would also really like this feature.

**Comment by znbaboy** [comment]
It would also be great to allow us to disable the badge altogether when implementing this!

**Comment by bkaankose** [comment]
> It would also be great to allow us to disable the badge altogether when implementing this!

You can disable the unread badge per-account under manage account -> account details.

**Comment by Zeuselhood** [comment]
Any news on the proposal? Its basicall the main reaseon why I don`t use Wino right now, since I will always have a 100+ Messages on the badge counter and i wouldnt notice when a new one comes in during the current session.

**Comment by bkaankose** [comment]
New version v2.0.13.0 is released to Microsoft Store yesterday. Closing as fixed. If this is still reproducible, please reopen with steps.
